app/styles/TilesetFinder.css

```diff
@@ -0,0 +1,249 @@
+@font-face {
+  font-family: 'FontAwesome';
+  src: url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/fonts/fontawesome-webfont.eot?v=4.7.0');
```

Comment:
What do we need fontawesome for? Could we just download and inline the icon? External references can lead to issues when being offline or within some secret organizations intranet.

Reply:
If some secret organizations' internet isn't connected they're going to have bigger fish to fry than the icons here.

But nevertheless, I included them as inlined SVGs.